### ISSUE TITLE
Combine blur and dirty validation

### ISSUE THREAD

**Issue by MortenHe** [formkit/formkit#973]
### Reproduction

https://formkit.link/7e1033c71366da383e945ca8e005bcef

### Describe the bug

In my example the first input has a required and min length validation rule.

I'd like to show the required message only when tabbing out the input without entering a value but also have a live validation for the min length when typing.

If i use "live" the required message is shown what I don't want.
If i use "dirty" no required message is shown when tabbing out of the empty input.
If i use "blur" there is no live checking the min length.

Any idea how to achieve this?

### Environment

• OS: windows
• Browser chrome

**Comment by MortenHe** [comment]
Is there a flag to check if the input was focused (but left without entering a value)?

Then I could maybe control the required display programmatically.

**Comment by justin-schroeder** [comment]
Yes, `context.state.blurred` is probably what you are looking for:

https://formkit.com/api-reference/context#state

**Comment by MortenHe** [comment]
Great, one last question

How can I access the state inside the FormKit element?

Can I set validation-visibility depending on the dirty flag of this input?

<FormKit type="search"
    :validation-visibility="state.dirty? 'live' : 'blur'"
....
>

**Comment by justin-schroeder** [comment]
After further reflection I think there is merit in having a combined value as another state flag.
